### PR TITLE
Make the button styling changes

### LINKED ISSUE #336: [design QA] button styling
I've mentioned this in a few styling tickets but it might warrant it's own ticket. 

Button text and line color should be "mobile blue" or white (when on a photo or color background).

All buttons should have the same font size, weight and have the same amount of padding.

Button hover state should fade to the inverse color like on https://mozilla-foundation-research.herokuapp.com/globalweb/ and https://advocacy.mozilla.org/

### PR DESCRIPTION
**Pull request by akshatnitd**
Fixes #336

### CODE REVIEW

**Comment by alanmoo** [comment]
Based on the conversation in #336, I think you made a change to mofo-bootstrap.css, but changes to that aren't reflected in this commit. That file is pulled in at build time, so a fix for this site can't be made in that file.

**Comment by akshatnitd** [comment]
@alanmoo Made the necessary amends to the commit. Kindly review.

File: app/scss/main.scss [hunk 1]
```diff
@@ -54,6 +54,10 @@ h6 {
   }
 }
 
+.btn {
```
Comment (alanmoo) [inline app/scss/main.scss]: This affects all buttons on the site. We only want to affect the ones on events pages like https://science.mozilla.org/programs/events/studygroup-call-july-15

Comment (alanmoo) [inline app/scss/main.scss]: This affects all buttons on the site. We only want to affect the ones on events pages like https://science.mozilla.org/programs/events/studygroup-call-july-15

Comment (akshatnitd) [inline app/scss/main.scss]: @alanmoo Made the changes only to the specific scss file

**Comment by akshatnitd** [comment]
@alanmoo Kindly review it. Made changes to specific scss files.

**Comment by akshatnitd** [comment]
@alanmoo Made the requested changes.

**Comment by alanmoo** [comment]
I think these are good, but I just realized the PR is against the develop branch. As per the ReadMe, I'm switching this to be against the master branch and will make sure things look good there.

**Comment by alanmoo** [comment]
Merging the master branch into this is causing some regression. What you actually need to do is rebase the changes you made onto master. This results in a fast-forward of your branch to catch up to master, then your changes are applied. However, there might be some conflicts here so the easiest thing to do is probably to cherry pick your changes and apply them to a new branch based on the latest `master`

**Comment by alanmoo** [comment]
And since `master` is using the newest version of MoFo Bootstrap, you might not need the new CSS, classNames could fix it all.

**Comment by akshatnitd** [comment]
@alanmoo Okay. Making the necessary changes :)

**Comment by akshatnitd** [comment]
Closing this. Opened a new PR #574

**Pull request closed without merging**